eng/deployment/product-construction-service-deploy.ps1

```diff
@@ -43,7 +43,7 @@ function StopAndWait([string]$pcsStatusUrl, [string]$pcsStopUrl) {
         } While ($pcsStateResponse.Content -notmatch "Stopped")
     }
     catch {
-        Write-Error "An error occurred: $($_.Exception.Message).  Deploying the new revision without stopping the service."
+        Write-Warning "An error occurred: $($_.Exception.Message).  Deploying the new revision without stopping the service."
```

Comment:
I changed this to warnings because the script was exiting after `Write-Error`. The other option would be to change the error preference, but I think this is better, as we will really stop if something catastrophic happens